proto/anki/collection.proto

```diff
@@ -30,7 +28,12 @@ message OpenCollectionRequest {
 }
 
 message CloseCollectionRequest {
+  message BackupPaths {
+    string col_path = 1;
```

Comment:
Since the Collection object should already know its own path, I think we probably don't need to pass it in when closing - I'd probably drop the inner message and pass backup_dir directly.

Reply:
(Collection opening already uses empty strings to represent optional args, so we could do that here too)